setup.py

```diff
@@ -124,6 +158,7 @@ def build_extensions(self):
         'License :: OSI Approved :: GNU General Public License v2 (GPLv2)',
         'Programming Language :: Python :: 3',
         'Operating System :: Microsoft :: Windows',
+        'Operating System :: MacOS :: MacOS X',
```

Comment:
Probably better to just use `'Operating System :: MacOS'`

Reply:
Done